ports/stm32/mboot/mboot_pack_dfu.py

```diff
@@ -86,9 +86,14 @@ def save(self):
 
     def load(self):
         with open(self.filename) as f:
-            self.sign_sk = self._load_data("mboot_pack_sign_secret_key", f.readline())
-            self.sign_pk = self._load_data("mboot_pack_sign_public_key", f.readline())
-            self.secretbox = self._load_data("mboot_pack_secretbox_key", f.readline())
+            lines = f.readlines()
+            if lines[0].startswith("//"):
```

Comment:
IMO this is a bit cryptic and inflexible.  Might be better to write a more general parser, eg:
```python
for line in f:
    for key, attr in (("mboot_pack_sign_secret_key", "sign_sk"), ...):
        if key in line:
            setattr(self, attr, self._load_data(key, line))
```

Reply:
Thanks, that's much more robust